test/integration/UpdatePublishersTest.js

```diff
@@ -0,0 +1,48 @@
+require('require-self-ref')
+const test = require('ava')
+const sinon = require('sinon')
+const Promise = require('bluebird')
+const config = require('~/src/config')
+const {createConsumer} = require('windbreaker-service-util/queue')
+const updatesPub = require('~/src/UpdatesPublisher')
+config.load()
+const producerOptions = {
+  queueName: 'test-queue'
+}
+updatesPub.configure(producerOptions, config.getAmqUrl(), 'www.notasite.fake', console)
+
+test.beforeEach('setup env', (t) => {
+  const consumerOptions = {
+    queueName: 'test-queue'
+  }
+
+  const sandbox = sinon.sandbox.create()
+
+  t.context = {
+    consumerOptions,
+    sandbox
+  }
+})
+
+test.afterEach('clean up', (t) => {
+  const {sandbox} = t.context
+  sandbox.restore()
+})
+
+test.serial('Will successfully detect changes and publish to rabbitmq', async (t) => {
```

Comment:
As I mentioned above, moving away from a singleton `UpdatesPublisher` would allow future tests that directly use the publisher (non proxyquired) be run in parallel.